EIPS/eip-2930.md

```diff
@@ -29,6 +29,15 @@ This EIP serves two functions:
 
 ## Specification
 
+### Definitions
+
+**`TransactionType`** `1`.  See [EIP-2718](./eip-2718.md)
+
+**`ChainId`** The transaction is valid only if this value is included in a block on a chain whose ID is equal to this value.
```

Comment:
Nitpick, it currently sounds like it doesn't actually _become_ valid until it's included in a block. Or even: until the `value` is included in a block
Maybe:
```suggestion
**`ChainId`** The transaction only valid on networks with this `chainID`.
```

Reply:
Applied, I like your wording better than mine.  😄